nodl_to_policy/policy.py

```diff
@@ -81,13 +81,14 @@ def get_profile(policy: etree._ElementTree, node_name: str) -> etree._ElementTre
         # namespace information not provided in NoDL description yet
         profile.attrib['ns'] = '/'
         profile.attrib['node'] = node_name
+        profile.attrib['ns'] = '/'
```

Comment:
Isn't this redundant with 2 lines above?

Reply:
Yes, this line is redundant. Fixed in b8a958c.